### PR TITLE
shippey

### PR DESCRIPTION
**Pull request by chocolatech**

### CODE REVIEW

File: Lesson 4 - DOM/code/homework/app.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 window.addEventListener('DOMContentLoaded', function () {
 
     //Create area and ship as `div` in `document` node
-    var playArea = {}, // ??
-        ship = {}; // ??
+    var playArea = document.createElement('div');
+    var ship = document.createElement('div');
```
Comment (senssei) [inline Lesson 4 - DOM/code/homework/app.js:5]: You do not need second `var` statement. :)

**Comment by senssei** [comment]
Very good. :+1:

File: Lesson 4 - DOM/code/homework/app.js (hunk 1 above)
Comment (piecioshka) [inline Lesson 4 - DOM/code/homework/app.js:5]: But is more readable @cobbie good job! 👍

Comment (senssei) [inline Lesson 4 - DOM/code/homework/app.js:5]: Lol. :+1:

**Pull request closed without merging**